server/scripts/datastore.js

```diff
@@ -0,0 +1,20 @@
+const { execSync } = require("child_process");
+require("dotenv").config();
+const process = require("process");
+const { env } = process;
+
+if (!env.DATASTORE_HOST || !env.DATASTORE_PORT || !env.DATASTORE_PROJECT_ID) {
+  console.error(`missing one of required config values`);
+  process.exit(1);
+}
+const command = [
+  `docker run -d`,
+  `-e "DATASTORE_PROJECT_ID=${env.DATASTORE_PROJECT_ID}"`,
+  `-e "DATASTORE_LISTEN_ADDRESS=0.0.0.0:8081"`,
+  `-h ${env.DATASTORE_HOST}`,
+  `-p ${env.DATASTORE_PORT}:8081`,
+  `--rm singularities/datastore-emulator`,
+].join(" ");
```

Comment:
I don't seem to understand whats going on here. Can you briefly explain?

Reply:
This is a utility script for running an instance of datastore locally using. E.g, when testing the server we don't want to be touching production data. It's optional to use this script, it's only for development purposes.